src/metabase/api/dashboard.clj

```diff
@@ -611,9 +622,18 @@
     (dashboard-card/delete-dashboard-cards! dashcard-ids)
     dashboard-cards))
 
+(defn- assert-no-invalid-dashboard-internal-dashcards [dashboard to-create]
```

Comment:
This function name took me a solid minute to understand. Maybe `assert-new-dashcards-are-not-internal-to-other-dashboards`? Or just add a comment

Reply:
I think the problem is that "dashboard-internal dashcards" make sense as a phrase, but the grouping is lost when the whole thing is kebab case